### PR TITLE
Add specialized class `File` for an input tag with type `file`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | -

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 - New #105: Add specialized class `File` for an input tag with type `file` (vjik)
 - Chg #105: Rename `Html::fileInput()` to `Html::file()`, `Html::file()` and `Input::file()` returns `File` (vjik)
+## 2.4.0 under development
```
Comment (samdark) [inline CHANGELOG.md]: Looks wrong.

**Comment by samdark** [comment]
Method rename means it could be released in major version only.

**Review by samdark — APPROVED** [review]

**Comment by vjik** [comment]
> Method rename means it could be released in major version only.

Modified code: now without BC-changes.

Plan:

**2.4.0**
- methods `Html::fileInput()` and `Input::file()` marked as deprecated.

**3.0.0**
- remove `Html::fileInput()`
- `Input::file()` will be return `File` instead of `Input`
- mark `Input::fileControl()` as deprecated

**4.0.0**
- remove  `Input::fileControl()`

File: src/Html.php [hunk 2]
```diff
@@ -789,13 +790,35 @@ public static function passwordInput(?string $name = null, $value = null, array
      * @param string|null $name The name attribute.
      * @param bool|float|int|string|Stringable|null $value The value attribute.
      * @param array $attributes The tag attributes in terms of name-value pairs.
+     *
+     * @deprecated In favour {@see file()}.
```
Comment (samdark) [inline src/Html.php:794]:
```suggestion
     * @deprecated In favour of {@see file()}.
```

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**